src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -4,21 +4,15 @@ import axios from 'axios';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
 
 import StorageUtils from '../../../utils/storage';
-import dimen from '../../../utils/dimen';
+import useViewPostTimeHook from './useViewPostTimeHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useViewPostTimeHook` is a good addition, as it helps to encapsulate the logic related to view post time in a separate hook, improving modularity and maintainability.
```diff
+ import useViewPostTimeHook from './useViewPostTimeHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed